src/Components/Sidebar/Sidebar.js

```diff
@@ -1,4 +1,4 @@
-import React, { useCallback } from "react";
+import React, { useCallback, useState } from "react";
 import "Components/Sidebar/Sidebar.scss";
```

Comment:
minor: why is components folder with a capital C? It's bothering me every time I see this repo's code